docs/openshift/kctlr-openshift-app-install.rst

```diff
@@ -51,82 +53,122 @@ Set up RBAC Authentication
 Deployments for Managing Routes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+```````````````````````````````
+
+The |kctlr| has a set of `Route configuration parameters`_ that you can add to any :code:`k8s-bigip-ctlr` Deployment if you want to manage Routes.
+
+See :ref:`Manage Routes with the BIG-IP Controller <kctlr-openshift-routes>` for instructions and examples.
 
-The |kctlr| has a set of `Route configuration parameters`_. See :ref:`Manage Routes with the BIG-IP Controller <kctlr-openshift-routes>` for examples and set-up instructions.
+.. _kctlr-openshift ha deploy:
 
-Deployments for BIG-IP HA
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Deployments for BIG-IP HA pairs/groups
+``````````````````````````````````````
 
-If you want to manage a BIG-IP HA pair or group, you'll need to deploy a |kctlr| instance for each device.
-See :ref:`BIG-IP High Availability in OpenShift <openshift deploy kctlr ha>` for more information.
+If you want to manage a BIG-IP HA pair or group, you'll need to deploy one |kctlr| instance per BIG-IP device.
+
+See :ref:`BIG-IP High Availability in OpenShift <openshift deploy kctlr ha>` for instructions and examples.
 
 .. _upload openshift deployment:
 
-Upload the Deployment
-`````````````````````
+Upload the Deployment to the OpenShift API Server
+-------------------------------------------------
+
 .. tip::
    :class: sidebar
 
-   For HA deployments, you can pass in each Deployment's filename in a single :command:`oc create` command.
+   If you're deploying multiple Controller instances in the same Project/namespace, you can pass each Deployment's filename in a single :command:`oc create` command.
+
+   Be sure to create all of the resources in the Project (namespace) that best suits your needs.
+
+   For example, if your Controller will manage multiple Projects, deploy the Controller and its supporting resources to :code:`-n kube-system`. This is the namespace in which all of the Kubernetes system controllers run.
+
+   If you intend to manage a single Project **and you are currently working in that Project**, you do not need to provide the :code:`-n` flag.
 
 Use the :command:`oc create` command to upload the Deployment to the OpenShift API server.
 
-.. code-block:: console
+.. parsed-literal::
 
-   oc create -f f5-k8s-bigip-ctlr_openshift-sdn.yaml
+   oc create -f **f5-k8s-bigip-ctlr_openshift-sdn.yaml** **[-n kube-system]**
    deployment "k8s-bigip-ctlr" created
 
-.. _kctlr verify pods:
+.. _kctlr openshift verify pods:
 
 Verify Pod(s)
-`````````````
+-------------
 
 You can verify that the Controller(s) created successfully using the :command:`oc get` command.
 
-You should see one :code:`k8s-bigip-ctlr` `Pod`_ for each Node in the Cluster. The example below shows one :code:`k8s-bigip-ctlr` Pod running in a test cluster with one worker node.
+You should see one :code:`k8s-bigip-ctlr` `Pod`_ for each Node in the Cluster. The example below shows one :code:`k8s-bigip-ctlr` Pod running in a test cluster that has one worker node.
```

Comment:
Only if we deployed via a DaemonSet would this be the case. Using a Deployment, with a replica setting of 1 will always just create one Pod.